pkg/cmd/workspace/code.go

```diff
@@ -86,12 +91,38 @@ var CodeCmd = &cobra.Command{
 			ideId = ideFlag
 		}
 
-		view_util.RenderInfoMessage("Opening the workspace in your preferred IDE")
+		view_util.RenderInfoMessage("Opening the workspace project '" + projectName + "' in your preferred IDE.")
```

Comment:
We prefer using `fmt.Sprintf` to string concatenation in our codebase so I'd like to ask you to adjust that.